controlplane/src/core/repositories/OrganizationRepository.ts

```diff
@@ -849,9 +851,40 @@ export class OrganizationRepository {
     return result[0];
   }
 
-  public deleteOrganization(organizationId: string) {
+  public deleteOrganization(organizationId: string, blobStorage: BlobStorage) {
     return this.db.transaction(async (tx) => {
       const oidcRepo = new OidcRepository(tx);
+      const fedGraphRepo = new FederatedGraphRepository(this.logger, tx, organizationId);
```

Comment:
If we go this road of deleting everything manuel, we need very good tests. Could you outline what is delete by constraints and what not?

Reply:
I have added a comment and a couple tests that cover what is deleted here